aspnetcore/grpc/browser.md

```diff
@@ -66,33 +50,27 @@ There is a JavaScript gRPC-Web client. For instructions on how to use gRPC-Web f
 
 ### Configure gRPC-Web with the .NET gRPC client
 
-The .NET gRPC client can be configured to make gRPC-Web calls. This is useful for [Blazor WebAssembly](xref:blazor/index#blazor-webassembly) apps, which are hosted in the browser and have the same HTTP limitations of JavaScript code. Calling gRPC-Web with a .NET client is [the same as HTTP/2 gRPC](xref:grpc/client), the only modification is how the channel is created.
+The .NET gRPC client can be configured to make gRPC-Web calls. This is useful for [Blazor WebAssembly](xref:blazor/index#blazor-webassembly) apps, which are hosted in the browser and have the same HTTP limitations of JavaScript code. Calling gRPC-Web with a .NET client is [the same as HTTP/2 gRPC](xref:grpc/client). The only modification is how the channel is created.
 
 To use gRPC-Web:
 
 1. Add a reference to the [Grpc.Net.Client.Web](https://www.nuget.org/packages/Grpc.Net.Client.Web) package.
 2. Configure the channel to use the `GrpcWebHandler`:
 
-```csharp
-// Configure a channel to use gRPC-Web
-var handler = new GrpcWebHandler(GrpcWebMode.GrpcWebText, new HttpClientHandler());
-var channel = GrpcChannel.ForAddress("https://localhost:5001", new GrpcChannelOptions
-    {
-        HttpClient = new HttpClient(handler)
-    });
+[!code-csharp[](~/grpc/browser/sample/Handler.cs?name=snippet_1)]
+
+The preceding code:
 
-// Create a client and make calls using the channel
-var client = Greeter.GreeterClient(channel);
-var response = await client.SayHelloAsync(new GreeterRequest { Name = ".NET" });
-```
+* Configures a channel to use gRPC-Web.
+* Creates a client and makes a call using the channel.
 
```

Comment:
The server must have been configured to support gRPC-Web.

change that to
The server must be configured to support gRPC-Web.